app/lib/theme/CustomDiffFields.tsx

```diff
@@ -11,7 +11,8 @@ const contentSuffixElement = (
 ): JSX.Element => {
   return (
     <span
-      id={id && `${id}-contentSuffix`}
+      // Add random number to id to avoid duplicate ids when multiple fields have the same contentSuffix
+      id={id && `${id}-contentSuffix-${Math.random().toFixed(2)}`}
```

Comment:
Even if fields have the same `contentSuffix`, shouldn't the `id`s be different? Are we accidentally repeating a field somewhere? I had this problem when both funding and emissions were displaying `holdbackPercentageToDate`, but that should be fixed now

Reply:
Yes, In this scenario, we seem to be duplicating the content suffix (`Duration: X month`) on the same line when the TEIMP start and end dates are already filled in.

Reply:
![Screenshot 2023-08-08 at 10 05 07 AM](https://github.com/bcgov/cas-cif/assets/52381975/7200dfd6-2db6-42b1-a63e-c58d36939500)

Reply:
Gotcha. That looks like a bug to me--if the end date changes, then the duration would also change. Unless someone changed both the start and end dates and ended up with the exact same amount of time, I guess. Maybe we can put to remove the random number in the bug card?

Reply:
In my opinion, it would be better to display the duration on one side only and avoid duplicating it. This would eliminate the need for a random ID generator.